telegram/ext/_application.py

```diff
@@ -630,6 +643,9 @@ def run_polling(
         If :attr:`post_shutdown` is set, it will be called after both :meth:`shutdown`
         and :meth:`telegram.ext.Updater.shutdown`.
 
+        If :attr:`post_stop` is set, it will be called between :meth:`stop` and
+        :meth:`telegram.ext.Updater.stop`.
+
```

Comment:
I think we should change these 3 paragraphs into:

``` python
"""
The order of execution by `run_polling` is roughly as follows:
- :meth:`initialize`
- *:meth:`post_init`*
- :meth:`updater.start_polling`
- :meth:`start`
...
- :meth:`updater.stop`
- :meth:`stop`
- *:meth:`post_stop`*
- :meth:`shutdown`
- *:meth:`post_shutdown`*
"""
```

Because it's much easier to visually see what will be executed when than reading it